docs/README.md

```diff
@@ -2,6 +2,8 @@
 
 SaltGUI is an open source web interface for managing a SaltStack server and its minions. Built using vanilla ES6 and implemented as a wrapper around the rest_cherrypy server a.k.a. salt-api.
 
+**🔒 Security Note**: For production deployments, SSL/TLS encryption is strongly recommended. See [SSL/TLS Configuration](#ssltls-configuration) for complete setup instructions.
```

Comment:
I'm not using icons (or unicode characters that are actually pictures) anywhere else in the documentation. can you please remove that one?